render.py

```diff
@@ -15,8 +16,8 @@
     loader = FileSystemLoader('templates'),
 )
 
-input_file = 'main.html'
-output_file = 'index.html'
+input_file = sys.argv[1]
```

Comment:
The current behaviour is not preserved. Could you please do the following? 

* update the README.md file to give the proper way to run `render.py`
* document the new arguments in render.py
* deal with incorrect numbers of arguments (exit and print help in this case)